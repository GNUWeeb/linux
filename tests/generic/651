#! /bin/bash
# SPDX-License-Identifier: GPL-2.0
# Copyright (c) 2021 Christian Brauner.  All Rights Reserved.
#
# FS QA Test 651
#
# Test that setattr works correctly.
#
. ./common/preamble
_begin_fstest auto attr cap idmapped mount perms

# Import common functions.
. ./common/filter

# real QA test starts here

_supported_fs generic
_require_idmapped_mounts
_require_test

echo "Silence is golden"

$here/src/idmapped-mounts/idmapped-mounts --test-setattr-fix \
	--device "$TEST_DEV" --mount "$TEST_DIR" --fstype "$FSTYP"

status=$?
exit
