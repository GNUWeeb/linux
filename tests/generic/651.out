QA output created by 651
Silence is golden
